maven/bnd-export-maven-plugin/src/test/resources/integration-test/test/export-from-dependencies/test.bndrun

```diff
@@ -1,4 +1,4 @@
--standalone: ${.}/index/index.xml.gz
+-standalone:
```

Comment:
Yes. For the maven plugins, all the bndrun files are standalone by definition. There is no workspace they can be a member of. The same is true for the non-workspace gradle task types.